### ISSUE TITLE
Markdown: Uncaught TypeError: Cannot read property 'replace' of null

### ISSUE THREAD

**Issue by davidbwaters** [directus/directus#1362]
I'm getting an error when adding a markdown field, saving the table options, and viewing the table.  The alert said report it to Marked issues, but it's the same error described in this thread, so I figured I would post it here.

Uncaught TypeError: Cannot read property 'replace' of null
https://github.com/chjj/marked/issues/678

**Comment by WellingGuzman** [comment]
Any step by step to reproduce? Unable to reproduce this issue.

**Comment by davidbwaters** [comment]
I created a new table, added a markdown field to it, saved it, and selected the table from the menu to view it.  After the table was created, I was able to add a markdown field.  I think there was an empty entry in the table after creation that could be causing it.

**Comment by davidbwaters** [comment]
Okay, sorry to be off-topic, but I believe the table-creation issue is the bug from [#1308](https://github.com/directus/directus/issues/1308) and will post my experience in that thread to try and help resolve that.  

The markdown problem seems to have something to do with the empty entry in the table upon creation.  It was the first table I created and I added the fields before viewing it.   After switching the field type to `textarea`,  I was able to access the table and noticed the empty entry. 

I am definitely a noob, but if I had to guess, maybe Marked needs an empty string set somewhere when markdown is selected?

Thanks a lot for all of the hard work.  Directus is a very impressive and refreshing approach to one of the oldest problems in web making!
